src/main/scala/no/ndla/articleapi/service/converters/contentbrowser/FilConverterModule.scala

```diff
@@ -26,12 +27,12 @@ trait FilConverterModule {
       val nodeId = content.get("nid")
       val importedFile = for {
         fileMeta <- extractService.getNodeFilMeta(nodeId)
-        audioPath <- attachmentStorageService.uploadFileFromUrl(nodeId, fileMeta)
-      } yield (HtmlTagGenerator.buildAnchor(audioPath, fileMeta.fileName, fileMeta.fileName), Seq(), ImportStatus(visitedNodes))
+        filePath <- attachmentStorageService.uploadFileFromUrl(fileMeta)
```

Comment:
Det ser sånn ut for min del. En måte å være helt sikker er å prefixe med `nodeId` slik det var før. fikser